test/scalarstats.jl

```diff
@@ -66,9 +66,9 @@ z2 = [8. 2. 3. 1.; 24. 10. -1. -1.; 20. 12. 1. -2.]
 @test zscore!(zeros(size(a)), a, [1, 2, 3], [0.5, 1.0, 2.0])    ≈ z1
 @test zscore!(zeros(size(a)), a, [1 3 2 4], [0.25 0.5 1.0 2.0]) ≈ z2
 
-@test zscore(a)    ≈ zscore(a, mean(a), std(a))
-@test zscore(a, 1) ≈ zscore(a, mean(a,1), std(a,1))
-@test zscore(a, 2) ≈ zscore(a, mean(a,2), std(a,2))
+@test zscore(a)    ≈ zscore(a, mean(a), std(a; corrected=false))
```

Comment:
NOTE: We could add that warning if we chose to make `corrected` a regular (vs keyword) argument. We'd just need to have a function which doesn't take the corrected argument, so that it can print a warning and call the function with the default setting? I initially thought making `corrected` a keyword argument was the best way to stay consistent with base, but `var`, `std` and `cov` aren't even consistent with each other in base, so it might make more sense to do whatever is best for our use case.

```
std(
std(A::AbstractArray; corrected, mean) in Base at statistics.jl:244         std(iterable; corrected, mean) in Base at statistics.jl:266
std(A::AbstractArray, region; corrected, mean) in Base at statistics.jl:263
julia> cov(
cov(x::AbstractArray{T,1} where T, corrected::Bool) in Base at statistics.jl:346
cov(x::AbstractArray{T,1} where T) in Base at statistics.jl:348
cov(X::AbstractArray{T,2} where T, vardim::Int64) in Base at statistics.jl:357
cov(X::AbstractArray{T,2} where T, vardim::Int64, corrected::Bool) in Base at statistics.jl:357
cov(X::AbstractArray{T,2} where T) in Base at statistics.jl:360
cov(x::AbstractArray{T,1} where T, y::AbstractArray{T,1} where T, corrected::Bool) in Base at statistics.jl:369
cov(x::AbstractArray{T,1} where T, y::AbstractArray{T,1} where T) in Base at statistics.jl:372
cov(X::Union{AbstractArray{T,1}, AbstractArray{T,2}} where T, Y::Union{AbstractArray{T,1}, AbstractArray{T,2}} where T, vardim::Int64) in Base at statistics.jl:382
cov(X::Union{AbstractArray{T,1}, AbstractArray{T,2}} where T, Y::Union{AbstractArray{T,1}, AbstractArray{T,2}} where T, vardim::Int64, corrected::Bool) in Base at statistics.jl:382
cov(x::AbstractArray{T,1} where T, Y::AbstractArray{T,2} where T) in Base at statistics.jl:385
cov(X::AbstractArray{T,2} where T, y::AbstractArray{T,1} where T) in Base at statistics.jl:386
cov(X::AbstractArray{T,2} where T, Y::AbstractArray{T,2} where T) in Base at statistics.jl:387